lib/_tls_wrap.js

```diff
@@ -1366,6 +1370,14 @@ Server.prototype.setTicketKeys = function setTicketKeys(keys) {
   this._sharedCreds.context.setTicketKeys(keys);
 };
 
+Server.prototype.getNumTickets = function getNumTickets() {
+  return this._sharedCreds.context.getNumTickets();
+};
```

Comment:
hmm.. I thought I had left a review comment on this previously but I'm not seeing it now... Stylistically, I'd much prefer these to get regular getter/setters (e.g. `server.numTickets = 1`) rather than separate functions like this.